nixos/modules/profiles/new-kernel.nix

```diff
@@ -0,0 +1,15 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  boot.kernelPackages = pkgs.linuxPackages_latest;
+  boot.supportedFilesystems = [ "bcachefs" ];
+  # Might be required as a workaround for bcachefs bug
+  # https://github.com/NixOS/nixpkgs/issues/32279#issuecomment-1093682970
+  boot.postBootCommands = ''
+    ${lib.getExe pkgs.keyutils "keyctl"} link @u @s
+  '';
```

Comment:
In keyctl's [man page](https://man7.org/linux/man-pages/man1/keyctl.1.html) it says:
```
       Session keyring: @s or -3
              Each process subscribes to a session keyring that is
              inherited across (v)fork, exec and clone. This is searched
              after the process keyring. Session keyrings can be named
              and an extant keyring can be joined in place of a
              process's current session keyring.

       User specific keyring: @u or -4
              This keyring is shared between all the processes owned by
              a particular user. It isn't searched directly, but is
              **normally linked to from the session keyring.**
```
So it looks like this might be a more general configuration that are not limited to bcachefs usage. But I'm not sure why we didn't link those 2 keyrings in the first place, and if that actually should be a general setting. So I'm just doing the safe thing as listed in the NixOS Wiki.